### PR TITLE
Update all the files and push to SB

### PR DESCRIPTION
**Pull request by lindsayplatt**
Forced all files to be pushed back up to SB so that they appear in the order I want them. This completes the work from the peer review and reconciliation. 

Note that some files were dropped (e.g. NLDAS meteo were reduced to just 2 zips) or file types changed (e.g. metrics files were converted from csv to feather). This is reflected in the change to `log/sb_posted_files.csv`.

### CODE REVIEW

File: log/sb_posted_files.csv [hunk 1]
```diff
@@ -1,30 +1,24 @@
...
-out_data/lake_temp_preds_GLM_GCM_ACCESS.zip,6206d3c2d34ec05caca53071,2023-02-14 04:55 CST
-out_data/lake_temp_preds_GLM_NLDAS.zip,6206d3c2d34ec05caca53071,2023-02-14 05:00 CST
-out_data/lake_temp_preds_EALSTM_NLDAS_MO-WY.zip,6206d3c2d34ec05caca53071,2023-02-14 05:08 CST
-out_data/lake_temp_preds_EALSTM_NLDAS_AR-MN.zip,6206d3c2d34ec05caca53071,2023-02-14 05:17 CST
-out_data/meteorological_inputs_NLDAS_8.zip,6206d3c2d34ec05caca53071,2023-02-14 05:23 CST
-out_data/meteorological_inputs_NLDAS_7.zip,6206d3c2d34ec05caca53071,2023-02-14 05:29 CST
-out_data/meteorological_inputs_NLDAS_6.zip,6206d3c2d34ec05caca53071,2023-02-14 05:37 CST
-out_data/meteorological_inputs_NLDAS_5.zip,6206d3c2d34ec05caca53071,2023-02-14 05:46 CST
-out_data/meteorological_inputs_NLDAS_4.zip,6206d3c2d34ec05caca53071,2023-02-14 05:55 CST
-out_data/meteorological_inputs_NLDAS_3.zip,6206d3c2d34ec05caca53071,2023-02-14 06:04 CST
```
Comment (AndyMcAliley) [inline log/sb_posted_files.csv:21]: Just to check: the fact that these lines are gone doesn't mean that the data are missing, does it?

**Review by AndyMcAliley — APPROVED** [review]
Looks good! I just have one clarifying question to understand why there are fewer files in the log of posted files now.

File: log/sb_posted_files.csv (hunk 1 above)
Comment (lindsayplatt) [reply log/sb_posted_files.csv:21]: Nope! I wrote this in my PR description but I was able to condense all the NLDAS meteo data into just 2 zips by de-duplicating them (see https://github.com/USGS-R/lake-temp-lstm-static-data-release/pull/78/commits/70d67c8d502a36bbb495fbd7af359eb53723e8f8 and https://github.com/USGS-R/lake-temp-lstm-static-data-release/pull/78/commits/043fac013a7405a93e607b901fa381d567ed6ab2). For some reason, there were duplicates of lots of records causing the NLDAS data files to be HUGE. I think it had something to do with pulling them from the GLM modeling repo and them being prepped for use and probably repeated _per lake_.

Comment (AndyMcAliley) [reply log/sb_posted_files.csv:21]: Oh, there it is, explained right up top! Sorry I missed that. All looks good!

**Pull request merged**